<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProgressWait</class>
 <widget class="QWidget" name="frmProgressWait">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QExtProgressWait" name="progressWait1" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QExtProgressWait" name="progressWait2" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="QExtProgressWait" name="progressWait3" native="true"/>
   </item>
   <item row="0" column="3">
    <widget class="QExtProgressWait" name="progressWait4" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QExtProgressWait" name="progressWait5" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QExtProgressWait" name="progressWait6" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="QExtProgressWait" name="progressWait7" native="true"/>
   </item>
   <item row="1" column="3">
    <widget class="QExtProgressWait" name="progressWait8" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QExtProgressWait</class>
   <extends>QWidget</extends>
   <header>qextProgressWait.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
